common-rest/src/main/java/org/uniprot/api/rest/output/converter/AbstractEntityIdHttpMessageConverter.java

```diff
@@ -25,7 +25,7 @@
 
     @Override
     public boolean canWrite(@Nullable Type type, Class<?> clazz, @Nullable MediaType mediaType) {
-        return this.canWrite(mediaType) && MessageConverterContext.class.isAssignableFrom(clazz);
+        return validMediaType(mediaType) && MessageConverterContext.class.isAssignableFrom(clazz);
```

Comment:
I think this method is never used. because we are not calling it from super class anymore